ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml

```diff
@@ -0,0 +1,83 @@
+                {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Health Check - Cluster", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default"]},
+                {"step": "gather-*", "failure_type": "pod_failure", "classification": "Must Gather - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
+                {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+                {"step": "openshift-pipelines-install", "failure_type": "all", "classification": "Operator Installation Failure", "group": {"name": "lp-tests", "priority": 1},"jira_additional_labels": ["!default"]},
+                {"step": "openshift-pipelines-tests", "failure_type": "all", "classification": "Test Failure", "jira_project": "SRVKP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"]},
+                {"step": "osd-*", "failure_type": "all", "classification": "Other", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
+                {"step": "rosa-cluster-deprovision", "failure_type": "all", "classification": "Infrastructure Deprovisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+                {"step": "rosa-cluster-provision", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+                {"step": "rosa-cluster-wait*", "failure_type": "all", "classification": "Infrastructure Provisioning - Rosa Cluster Operators", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+                {"step": "rosa-sts-account-roles-create", "failure_type": "all", "classification": "Account Roles Creation - AWS", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2},"jira_additional_labels": ["!default"]},
+                {"step": "rosa-sts-account-roles-delete", "failure_type": "all", "classification": "Account Roles Deletion - AWS", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
+                {"step": "rosa-conf-idp-htpasswd", "failure_type": "all", "classification": "Admin Access - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]}
+              ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.15-lp","rosa-hypershift-lp","pipelines-lp"]'
```

Comment:
```suggestion
      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/rosa/lp-interop.json
      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.15-lp","rosa-hypershift-lp","pipelines-lp"]'
```

Reply:
+1

Reply:
@oharan2 Is the environmental variable FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS necessary? Is it always in the same location?